pyproject.toml

```diff
@@ -1,2 +1,32 @@
-[tool.black]
-required-version = "22"
+[tool.ruff]
+exclude = [
+    "migrations",
+]
+
+line-length = 88
+
+target-version = "py311"
+
+lint.select = [
+    "E",   # pycodestyle errors
+    "W",   # pycodestyle warnings  
+    "F",   # pyflakes (critical errors only)
+]
+
+lint.ignore = [
+    "E501",   # line too long
+    "E722",   # bare except
+    "E731",   # lambda assignment
+    "F401",   # unused imports
+    "F403",   # star import (unable to detect undefined names from star imports)
+    "F405",   # `env` may be undefined, or defined from star imports
+    "F841",   # unused variable
```

Comment:
Is this to avoid the no-self-use pylint lines? I do think it's helpful to flag unused variables left in by mistakes